CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# v3.1.0 (January 5, 2024)
+ * [#42](https://github.com/sourcetoad/react-native-fs2/pull/42) - Switch to NPM from Yarn
+ * [#41](https://github.com/sourcetoad/react-native-fs2/pull/41) - Update example app to RN 0.73.x
+ * [#35](https://github.com/sourcetoad/react-native-fs2/pull/35) - Update example app to RN 0.72.x
+ * Automatic updates via Dependabot
```

Comment:
Wasn't sure how much detail to go into about these

Reply:
theres so many - I guess as an end user it would be cool to know exacts, but as writer of it - this is good